content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
+configure with `dvc remote`. DVC uses them to store and fetch large files that
+don't normally fit inside Git repos.
+
+DVC needs both kinds of remotes for backing up and sharing experiments.
+
+Experiment files that are normally tracked in Git (like code versions) are
+shared using Git remotes, and files or directories tracked with DVC (like
+datasets) are shared using DVC remotes.
+
+> See [Git remotes guide] and `dvc remote add` for information on setting them
+> up.
+
+[git remotes guide]:
+  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+
+Normally, there should already be a Git remote called `origin` when you clone a
+repo. Use `git remote -v` to list your Git remotes:
+
+```dvc
+$ git remote -v
+origin  https://github.com/iterative/get-started-experiments (fetch)
+origin  https://github.com/iterative/get-started-experiments (push)
+```
+
+Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+
+```dvc
+$ dvc remote list
+storage https://remote.dvc.org/get-started-experiments
+```
+
+_Q: What about neither, `cache: false` objects, and objects tracked both by DVC
+and Git?_
+
+## Uploading experiments to remotes
+
+You can upload an experiment and its files to both remotes using `dvc exp push`
+(requires the Git remote name and experiment name as arguments).
+
+```dvc
+$ dvc exp push origin exp-abc123
+```
+
+> Use `dvc exp show` to find experiment names.
+
+This pushes the necessary DVC-tracked files from the cache to the default DVC
+remote (similar to `dvc push`). You can prevent this behavior by using the
+`--no-cache` option to the command above.
+
+If there's no default DVC remote, it will ask you to define one with
+`dvc remote default`. If you don't want a default remote, or if you want to use
+a different remote, you can specify one with the `--remote` (`-r`) option.
+
+DVC can use multiple threads to upload files (4 per CPU core by default). You
+can set the number with `--jobs` (`-j`). Please note that increases in
+performance also depend on the connection bandwidth and remote configurations.
+
+DVC has a mechanism called the [run-cache] that stores the artifacts from
+intermediate stages. For example, if there is an intermediate step that applies
+data-augmentation on your dataset and you would like to push these artifacts as
+well as the end products of the experiments, you can use `--run-cache` flag to
+push all of these to the DVC remote. `--run-cache` flag pushes all artifacts
+referenced in `dvc.lock` file.
+
+[run-cache]: /doc/user-guide/project-structure/internal-files#run-cache
+
+## Listing experiments remotely
+
+In order to list experiments in a DVC project, you can use the `dvc exp list`
+command. With no command line options, it lists the experiments in the current
+project.
+
+You can supply a Git remote name to list the experiments:
+
+```dvc
+$ dvc exp list origin
+main:
+	cnn-128
```

Comment:
💅 Quick note: you're using tab characters here. I think we use spaces in existing docs but I'm not sure. I guess as long as the doc is consistent internally (there were some spaces later but I changed them to tabs).

Reply:
I should have copied from the terminal but I'd rather change to spaces.

Reply:
Replaced with spaces. Thank you :)

Reply:
OK, thanks.